cpp/tests/io/parquet_reader_test.cpp

```diff
@@ -2187,6 +2188,52 @@ TEST_F(ParquetReaderTest, StringsWithPageStats)
   }
 }
 
```

Comment:
Would it be possible to move this test to its own file in the `cpp/tests/large_strings` folder and use the fixture there which has some code to reuse large strings columns?
This way, we can control the memory usage and environment a little better.
I'm open to making more helpful utilities there as well.
Here is an example usage which may help: https://github.com/rapidsai/cudf/blob/67d427deb3cf18d1139b76aecc1e6a3e9d5253f3/cpp/tests/large_strings/concatenate_tests.cpp#L27-L29

Reply:
I'll give it a whirl.

Reply:
@davidwendt I moved it, but I'm not currently using anything from the fixture other than the `large_strings_enabler`. Can you take a look and make any suggestions you can think of?